### PR TITLE
Edits to administration logging page

### PR DESCRIPTION
**Pull request by bobadair**
Edits to administration/logging.md file.

Signed-off-by: Bob Adair <bob.github@att.net>

### CODE REVIEW

File: administration/logging.md [hunk 1]
```diff
@@ -24,9 +24,9 @@ Log files are written to either `userdata/log` (manual setup) or `/var/log/openh
 
 ## Karaf Console
 
-The [Karaf console](console.html) allows to monitor the log in real-time.
+The [Karaf console](console.html) allows you to monitor the log in real-time.
```
Comment (Confectrician) [inline administration/logging.md:27]:
```suggestion
The [openHAB console](console.html) allows you to monitor the log in real-time.
```

**Review by Confectrician — COMMENTED** [review]
Thanks. 👍 

One small comment below.

**Comment by Confectrician** [comment]
LGTM

anyting you want to improve or can i merge it @bobadair ?

**Comment by bobadair** [comment]
No, I think the conflicts are all fixed now. It looks ok to me. Please just check to make sure I didn't mess anything up when I resolved them. :-)

**Comment by Confectrician** [comment]
https://deploy-preview-1345--openhab-docs-preview.netlify.app/docs/administration/logging.html

looks good to me .

**Review by Confectrician — APPROVED** [review]

**Pull request merged**